lxd/daemon.go

```diff
@@ -220,7 +220,7 @@ func allowAuthenticated(d *Daemon, r *http.Request) response.Response {
 }
 
 // allowProjectPermission is a wrapper to check access against the project, its features and RBAC permission
-func allowProjectPermission(feature string, permission string) func(d *Daemon, r *http.Request) response.Response {
+func allowProjectPermission(_ string, permission string) func(d *Daemon, r *http.Request) response.Response {
```

Comment:
@stgraber this argument isnt used, shall I remove it, or should it be used?

Reply:
I don't think we've ever had a need for it but may come in later so shouldn't hurt to keep it and pass the feature name through in calls.

Reply:
@stgraber ok will drop this commit to keep the arg named.